framework/include/executioners/SolveObject.h

```diff
@@ -41,7 +41,9 @@ class SolveObject : public MooseObject, public PerfGraphInterface, public Postpr
   /// Reference to FEProblem
   FEProblemBase & _problem;
   /// Displaced problem
-  std::shared_ptr<DisplacedProblem> _displaced_problem;
+  /// We would rather have a shared pointer here. But the displaced problem currently must be destructed
+  /// before the displaced mesh, and keeping a shared pointer here delays this
```

Comment:
I don't know that we would rather have a shared pointer here. I don't see any reason why any `SolveObject` (there could be many of them) should be considered an owner of the displaced problem

Reply:
yeah this is outdated. I ll delete the comment